setup.py

```diff
@@ -80,6 +86,103 @@ class Components:
 else:
     from setuptools.command.build_ext import build_ext
 
+music_home = ""
+if Components.MUSIC:
+    """
+    If music is not installed, try to install it. In principle, a dynamic
```

Comment:
this shouldn't be part of the setup.py logic.

Reply:
(the installation shall come from the docker image)

Reply:
For macOS we need to build MUSIC and store in on Azure as secure filezip (like we do for readline)

Reply:
I accept this. I didn't (and don't fully) understand how to carry it out.

Reply:
I can take point